jumanji/__init__.py

```diff
@@ -142,3 +142,10 @@
 # LevelBasedForaging  with a random generator with 8 grid size,
 # 2 agents and 2 food items and the maximum agent's level is 2.
 register(id="LevelBasedForaging-v0", entry_point="jumanji.environments:LevelBasedForaging")
+
+###
+# Swarm Environments
+###
+
+# Search-and-Rescue environment
+register(id="SearchAndRescue-v0", entry_point="jumanji.environments:SearchAndRescue")
```

Comment:
Is it worth registering the environment with different vision models?

Reply:
Do you mean fully versus partially observable?

Reply:
Ah I think you mean the different observation functions. I'd say we should aim to use the observation that only visualizes targets and searchers within their vision cones. If we see a good training curve with this then that should be the default.

In general I'd like to have a set of scenarios for most/all environments in jumanji (see #248). So it would be cool to think of a set (3-4) easy/hard envs and we can register those. If they happen to have different observation models that's fine

Reply:
Yeah I was picturing something along these lines, like the easy one is where un-found targets are visible, and then harder versions use the version with hidden targets.